### PR TITLE
Am17072020

### PR DESCRIPTION
**Pull request by MotinhaAna**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #< insert number here >.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by sushi2k** [comment]
Hi @luisfontes19. Can you please do a QA on this so we can merge it? Thanks

File: Document-ptpt/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 1]
```diff
@@ -1,49 +1,50 @@
...
+Informação privada, neste contexto, diz respeito não só às credenciais do utilizador mas também a outros dados sensíveis, como por exemplo:
 
-- Personally identifiable information (PII) that can be abused for identity theft:  Social security numbers, credit card numbers, bank account numbers, health information;
-- Highly sensitive data that would lead to reputational harm and/or financial costs if compromised: Contractual information, information covered by non-disclosure agreements, management information;
-- Any data that must be protected by law or for compliance reasons.
+- Informação de Identificação Pessoal (Personally identifiable information, PII)) que pode ser abusada para roubo de identidade, o que inclui: números de segurança social, números de cartões de crédito, números de conta bancária, ou mesmo dados identificadores de estados de saúde;
+- Informação altamente sensível que pode pôr em causa a reputação do indivíduo, e/ou o estado financeiro do mesmo, se comprometida: Informação de não divulgação, ou outra informação de gestão individual;
+- Quaisquer dados que devem ser protegidos por lei ou outras razões.
 
-## Security Verification Requirements
```
Comment (luisfontes19) [inline Document-ptpt/0x07-V2-Data_Storage_and_Privacy_requirements.md:17]: You need to remove this line (markdown linter)

**Review by luisfontes19 — CHANGES_REQUESTED** [review]
https://owasp.org/www-project-mobiletop-10/2016-risks/m10-extraneous-functionality is down, check if it was already replaced in recent commits


There are a few comments without suggestions, so be sure to go through them 😊

**Comment by sushi2k** [comment]
Hi @luisfontes19. Can you please check if everything is fixed and approve if ready? Thanks

**Comment by luisfontes19** [comment]
there are still two suggested changes pending an action...
@MotinhaAna can you check them?

**Comment by sushi2k** [comment]
Hi @MotinhaAna. Thanks for the PR! Can you please have a look at the suggestions that are pending?

**Review by luisfontes19 — APPROVED** [review]

**Review by sushi2k — APPROVED** [review]
Thanks everybody for the PR!

**Pull request merged**